src/examples/regression_test.json

```diff
@@ -1,14 +1,14 @@
 {
   "Voting_": {
-    "digest": "be2fb91778c112dba345230d33f53b60d6aebc9d21c34af42d2d06169c03521",
+    "digest": "1ea9c9b3a7a4a496b7ff475c22703afc1d76e1d43a083f158a8e0fcdf5c38299",
     "methods": {
       "voterRegistration": {
-        "rows": 1259,
-        "digest": "252726e585fdd28607537e32214069ab"
+        "rows": 1260,
```

Comment:
that's interesting, I wonder where that extra constraint comes from 🤔

Reply:
One more constant that wasn't used before takes one half generic gate!